profile-controller/rockcraft.yaml

```diff
@@ -0,0 +1,51 @@
+# Constructed from https://github.com/kubeflow/kubeflow/blob/v1.7-branch/components/profile-controller/Dockerfile
+
+name: profile-controller
+summary: Controller for Kubeflow Profile objects
+description: Controller for Kubeflow Profile objects
+version: v1.7.0_22.04_1 # version format: <KF-upstream-version>_<Charmed-KF-version>
+
+# TODO: use base: bare
+base: ubuntu:22.04
+build-base: ubuntu:22.04
+
+license: Apache-2.0
+
+platforms:
+  amd64:
+
+services:
+  profile-controller:
+    override: replace
+    command: /manager
+    startup: enabled
+
+parts:
+  profile-controller:
+    plugin: go
+    source-type: git
+    source: https://github.com/kubeflow/kubeflow.git
```

Comment:
Need to specify branch or a tag, eg. `source-tag: v1.7.0-branch`

Reply:
Added